Mathlib/NumberTheory/KummerDedekind.lean

```diff
@@ -343,4 +357,29 @@ theorem Ideal.irreducible_map_of_irreducible_minpoly (hI : IsMaximal I) (hI' : I
   rw [Multiset.attach_map_val, Multiset.map_singleton, Subtype.coe_mk]
   exact normalizedFactors_irreducible hf
 
+open Set Classical in
+/-- Let `Q` be a lift of factor of the minimal polynomial of `x`, a generator of `S` over `R`, taken
+`mod I`. Then (the reduction of) `Q` corresponds via
+`normalizedFactorsMapEquivNormalizedFactorsMinPolyMk` to
+`span (I.map (algebraMap R S) ∪ {Q.aeval x})`.  -/
+theorem normalizedFactorsMapEquivNormalizedFactorsMinPolyMk_symm_apply_eq_span
```

Comment:
Is this the longest name in mathlib?

Reply:
I can change it to `normalizedFactorsMapEquivNormalizedFactorsMinPolyMk_symm_apply`, but to make it shorter we need to change the name of `normalizedFactorsMapEquivNormalizedFactorsMinPolyMk` (happy to do it if you have a suggestion)